quill-sql/src/main/scala/io/getquill/context/sql/SqlQuery.scala

```diff
@@ -21,8 +21,6 @@ sealed trait SqlQuery {
     import io.getquill.idiom.StatementInterpolator._
     implicit val naming = Literal
     implicit val tokenizer = defaultTokenizer
-    implicit def liftTokenizer: Tokenizer[Lift] =
-      Tokenizer[Lift](_ => stmt"?")
```

Comment:
that was not used at all, I have no idea why. However users actually uses `toString` from `Ast` class, and hence it's safe to remove this.